plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -53,9 +53,13 @@ const DefaultAPITokenMountPath = "/var/run/secrets/kubernetes.io/serviceaccount"
 const PluginName = "ServiceAccount"
 
 func init() {
-	admission.RegisterPlugin(PluginName, func(client clientset.Interface, config io.Reader) (admission.Interface, error) {
+	admission.RegisterPlugin(PluginName, func(client clientset.Interface, config io.Reader, stopCh chan struct{}) (admission.Interface, error) {
 		serviceAccountAdmission := NewServiceAccount(client)
-		serviceAccountAdmission.Run()
+		if stopCh == nil {
```

Comment:
I would prefer to have one `Run` function which has a `stopCh`. This is confusing.